frontend/src/pages/team/Devices/index.vue

```diff
@@ -38,7 +38,7 @@
                     </ff-button>
                 </div>
             </template>
-            <template v-if="devices.length === 0">
+            <template v-if="this.devices.size === 0">
```

Comment:
There's no native wrapper, but there's no reason we couldn't add one. The change was needed as `this.devices` is no longer an array, instead it's a `Map`, and `Map` uses size rather than length to count the number of objects.